### PR TITLE
FHG-2862 Add tests for contact methods page

### PR DESCRIPTION
**Pull request by harshatalent**
### Jira link

[FHG-2862](https://dfedigital.atlassian.net/browse/FHG-2862)

- Added tests for contact methods page - howCanServiceContactPage.spec.js
- Added support comands
- Modified other page specs to use test url

### CODE REVIEW

File: cypress/e2e/01-referral-service/PageSpecs/cannotConnectFamilyPage.spec.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 describe('Cannot connect family to service page - FHG-2853', ()=> {
 	beforeEach(()=> {
-		cy.visit('https://dev.connect-families-to-support.education.gov.uk');
+		cy.visit('https://test.connect-families-to-support.education.gov.uk');
```
Comment (mdolaDFE) [inline cypress/e2e/01-referral-service/PageSpecs/cannotConnectFamilyPage.spec.js:3]: Can we use cy.visit('/') instead , so the base Url gets picked from the config set up. 
When running UI mode we can use custom commands yarn run open:referral-dev / yarn run open:referral-test

**Review by mdolaDFE — APPROVED** [review]
LGTM , just a minor change on the cy.visit('/') - can we make these changes on all the page specs please ?

**Review by mdolaDFE — APPROVED** [review]

File: cypress/e2e/01-referral-service/PageSpecs/cannotConnectFamilyPage.spec.js (hunk 1 above)
Comment (harshatalent) [reply cypress/e2e/01-referral-service/PageSpecs/cannotConnectFamilyPage.spec.js]: Changed base url in config.js. Also, updated spec files to use base url or pick the url from custom commands

**Review by mdolaDFE — APPROVED** [review]
👍

**Pull request merged**